Content.Server/NPC/Systems/Faction/NPCCombatTargetSystem.cs

```diff
@@ -0,0 +1,73 @@
+using Content.Shared.Damage;
+using Content.Shared.Mobs.Components;
+using Content.Server.NPC.Components;
+using Content.Server.Destructible;
+using Robust.Shared.Timing;
+
+namespace Content.Server.NPC.Systems
+{
+    public sealed class NPCCombatTargetSystem : EntitySystem
```

Comment:
is this code meant to be included here? looks not immediately relevant to the faction stuff though i get what its supposed to be

Reply:
the entire PR is adding hostility/friendliness outside of faction so it's in scope I think